nengo/networks/tests/test_actionselection.py

```diff
@@ -49,7 +49,6 @@ def test_thalamus(Simulator, plt, seed):
     assert output[0] > 0.8
     assert np.all(output[1:] < 0.01)
 
-
 if __name__ == "__main__":
-    nengo.log(debug=True)
-    pytest.main([__file__, '-v'])
+    import sys
+    pytest.main(sys.argv)
```

Comment:
I'm okay taking them out if that's the consensus. If that's what people want, I'll write up a page in the docs with more details on running unit tests...